pkg/api/config/config.go

```diff
@@ -68,8 +68,6 @@ type HTTPConfig struct {
 	Auth             *AuthConfig
 	RawAccessControl map[string]interface{} `mapstructure:"accessControl,omitempty"`
 	Realm            string
-	AllowReadAccess  bool             `mapstructure:",omitempty"`
```

Comment:
I think we shouldn't remove them for now, only throw an warning if they are not nil, telling clients that these options are deprecated and to use AccessControl instead, also we need to document the new field in examples/README.md